src/filters/regex_manager.rs

```diff
@@ -127,7 +127,7 @@ impl RegexManager {
     }
 }
 
-#[cfg(test)]
+#[cfg(feature = "debug-info")]
```

Comment:
should be `#[cfg(all(test, feature = "debug-info"))]`

Reply:
Done